x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts.ts

```diff
@@ -75,15 +75,29 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await loggingMessageInput.clearValue();
       await loggingMessageInput.type('test message');
 
-      await testSubjects.click('slackAddVariableButton');
-      const variableMenuButton = await testSubjects.find('variableMenuButton-0');
-      await variableMenuButton.click();
+      // TODO: uncomment variables test when server API will be ready
+      // await testSubjects.click('slackAddVariableButton');
+      // const variableMenuButton = await testSubjects.find('variableMenuButton-0');
+      // await variableMenuButton.click();
```

Comment:
Has this been forgotten or is it still in progress? :)

Reply:
In progress, we have an issue for this https://github.com/elastic/kibana/issues/58529